include/nuttx/binfmt/binfmt.h

```diff
@@ -97,6 +97,7 @@ struct binary_s
 
   uint8_t priority;                    /* Task execution priority */
   size_t stacksize;                    /* Size of the stack in bytes (unallocated) */
+  uid_t proguid;                       /* User identity */
```

Comment:
How does this proguid get set?

Reply:
The apps side commit need follow.
https://github.com/apache/nuttx-apps/pull/1691/